cfgov/jinja2/v1/owning-a-home/explore-rates/index.html

```diff
@@ -587,15 +587,10 @@ <h4 class="next-steps-heading">Save for a larger down payment.</h4>
                 </div>
                 <section id="about" class="about wrapper">
                     <div class="content-l">
-                        <div class="about-data content-l_col content-l_col-2-3">
-                            {% include "_templates/ratings-form.html" %}
-                        </div>
-                        <div class="content-l_col content-l_col-1-3">
-                            <div class="o-form">
-                                <h3 class="subhead">About our data source for this tool</h3>
-                                <p>The lenders in our data include a mix of large banks, regional banks, and credit unions. The data is updated semiweekly every Wednesday and Friday at 7 a.m. In the event of a holiday, data will be refreshed on the next available business day.</p>
-                                <p>The data is provided by Informa Research Services, Inc., Calabasas, CA. <a id="informars" href="https://www.informars.com" target="_blank" rel="noopener noreferrer">www.informars.com</a>. Informa collects the data directly from lenders and every effort is made to collect the most accurate data possible, but they cannot guarantee the data’s accuracy.</p>
-                            </div>
+                        <div class="content-l_col content-l_col-1">
+                            <h3 class="subhead">About our data source for this tool</h3>
+                            <p>The lenders in our data include a mix of large banks, regional banks, and credit unions. The data is updated semiweekly every Wednesday and Friday at 7 a.m. In the event of a holiday, data will be refreshed on the next available business day.</p>
+                            <p>The data is provided by Informa Research Services, Inc., Calabasas, CA. <a id="informars" href="https://www.informars.com" target="_blank" rel="noopener noreferrer">www.informars.com</a>. Informa collects the data directly from lenders and every effort is made to collect the most accurate data possible, but they cannot guarantee the data’s accuracy.</p>
```

Comment:
I don't see how the `id="informars"` is used. Could probably be removed. Though doesn't do harm staying either (other than clutter). Could also be done in separate PR after this one to isolate any issues.